features/step_definitions/gegeven-stepdefs-persoon.ts

```diff
@@ -0,0 +1,25 @@
+import { Given } from '@cucumber/cucumber';
+import { Persoon } from './brp/persoon-entity';
+import { Aanduiding } from './support/aanduiding';
+
+Given('de persoon {string}', function (persoonAanduiding: string) {
+    if(!this.context.personen) {
+        this.context.personen = {};
+    }
+    this.context.personen[persoonAanduiding] = new Persoon(undefined, undefined, persoonAanduiding);
+    this.huidigAanduiding = Aanduiding.persoon(persoonAanduiding);
+});
+
+Given('met A-nummer {string}', function (aNummer: string) {
```

Comment:
Kan een {word} zijn.

```suggestion
Given('met A-nummer {word}', function (aNummer: string) {
```